pyat/at/lattice/lattice_object.py

```diff
@@ -217,3 +224,111 @@ def ff(elem):
 
         self._radiation_switch(repfunc(cavity_pass), repfunc(dipole_pass), repfunc(quadrupole_pass))
         self._radiation_on = False
+
+    def linopt(self, *args, **kwargs):
+        """
```

Comment:
I would rather the same docstring to avoid them getting out of sync. Something like this, although it's a little bit tricky:

```
    def linopt(self, *args, **kwargs):
        """See at.physics.linear.linopt().
        """
        ...
    Lattice.linopt.__func__.__doc__ += linopt.__doc__
```

Reply:
I was looking for a way to do that ! Thanks